api/test/integration/env/configurations/base/manager/master_only/agent_info_to_wdb.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
```

Comment:
Why are we creating a .sh file to run a .py? .py files are also run in the entrypoint

Reply:
The `.py` files, which are run in the entrypoint, use the system python interpreter:

```
for py_file in /configuration_files/*.py; do
  /usr/bin/python3 $py_file
```
And there are some dependencies which are not installed there:

```
/usr/bin/python3 update_agent_info.py 
Traceback (most recent call last):
  File "update_agent_info.py", line 3, in <module>
    import yaml
ModuleNotFoundError: No module named 'yaml'

```